Source/WebKit/Scripts/generate-plain-cpp-swift-interop-header.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+# Swift/C++ interop generates a header file to be included in C++ code that provides
+# definitions of types encountered in Swift.
+# There's built-in Xcode functionality to generate this file.
+# But we need to do it twice:
+# - once, processing Swift code that pertains to objective-C[++] types to generate
+#   a header for inclusion in objective-C++ code
+# - once, processing only Swift code that does not involve objective-C[++ types,
+#   to generate a header which can be safely included in pure C++ code.
+# This shell script does the second thing. We use the built-in XCode action
+# for the first.
+# Working around rdar://152836730
```

Comment:
Not sure I fully understand the purpose of this script, but can it be removed entirely once this bug is fixed in our toolchains?